pages/common/tuckr.md

```diff
@@ -0,0 +1,24 @@
+# tuckr
+
+> Dotfile manager written in Rust.
+> More information: <https://github.com/RaphGL/Tuckr>.
+
+- Check dotfile status:
+
+`tuckr status`
+
+- Add all dotfiles to system:
+
+`tuckr add \*`
+
+- Add all dotfiles except specified programs:
+
+`tuckr add \* -e program1,program2`
```

Comment:
```suggestion
`tuckr add \* -e {{program1}},{{program2}}`
```
Note: `{{` and `}}` won't be displayed in user side it is used in our clients to highlight command arguments.